src/MediaFileReader.js

```diff
@@ -219,6 +219,15 @@ class MediaFileReader {
 
     return size;
   }
+
+  getUnsignedInteger32At(offset: number): number {
```

Comment:
This should be only `getInteger32At`, the scheme being used is:
`getS<type>` for signed and `get<type>` for unsigned.

Having said that, I do prefer the more explicit naming you have here. But more importantly I want the convention to be consistent.